pkg/wireguard-tools/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM linuxkit/alpine:3744607156e6b67e3e7d083b15be9e7722215e73 as tools
+RUN echo http://dl-cdn.alpinelinux.org/alpine/edge/main > /etc/apk/repositories && \
+    echo http://dl-cdn.alpinelinux.org/alpine/edge/testing >> /etc/apk/repositories && \
+    apk update && \
```

Comment:
I can play with this, but I'm not sure it will work as we want: wireguard-tools is in `edge/testing`, whereas tools/alpine uses a release, and things could get funky mixing and matching. Will play though.

(Maybe we want to merge this PR as is, and I'll submit subsequent ones for doing different things with wireguard-tools?)